.github/workflows/ci.yml

```diff
@@ -35,19 +39,19 @@ jobs:
         curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/download/${KREW_VERSION}/krew.{tar.gz,yaml}"
         tar zxvf krew.tar.gz
         ./krew-linux_amd64 install --manifest=krew.yaml --archive=krew.tar.gz
-        
+
     - name: Install validate-krew-manifest plugin
       run : |
         set -euo pipefail
         export GOBIN=$HOME/bin
         go get sigs.k8s.io/krew/cmd/validate-krew-manifest@${KREW_VERSION}
-      
+
     - name: Validate updated plugin manifests
       run : |
         set -x
         set -euo pipefail
         export PATH="${KREW_ROOT:-$HOME/.krew}/bin:$PATH"
-        git diff --name-only --diff-filter=AM ${{ github.sha }} -- plugins/ |
+        git diff --name-only --diff-filter=AM origin/master ${{ github.sha }} -- plugins/ |
```

Comment:
I think this will save the day but i don’t think it’s the ultimate solution.

GH should be providing us what the base branch is. 
This wouldn’t work if we had multiple branches for some reason and sent PRs to them.

Reply:
Quick search reveals https://stackoverflow.com/a/57862507/54929 env vars like  "$GITHUB_BASE_REF..$GITHUB_SHA" could these be useful?

Reply:
It could be useful, but I don't know how Github will get the difference.

I mean I already tried this one, but the problem I found:
The local repository in your build doesn't contain the commits you're referencing between original and forked repo, then it will through `fatal: bad object` or `fatal: bad revision` as [here](https://github.com/alonyb/krew-index/runs/435257792?check_suite_focus=true#step:7:18)

Maybe using something like [this](https://github.com/bdougie/fetch-upstream)  to fetch repos, my concerns appear from [here](https://stackoverflow.com/questions/25192708/how-can-i-compare-my-local-forked-repository-with-changes-that-may-have-been-mad)

It just a concern maybe I am wrong.

Reply:
Good point. Let's see if the modified version does the trick.

Reply:
Ok, looks good. If you have had a look, I'll remove the broken manifest so that we can merge.